src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -142,6 +142,16 @@ public int parseArguments(Parameters params) throws CmdLineException {
   )
   String collidingProvidesFile = null;
 
+  @Option(
+    name = "--tracer_mode",
+    hidden = true,
+    usage =
+        "Shows the duration of each compiler pass and the impact to "
```

Comment:
Interesting, I have never used this. Does it have any other documentation you can point off from there (i.e. official closure docs)?

Reply:
Done